### DISCUSSION TITLE
Editor: Selecting timing points shouldn't jump to them on the timeline

### DISCUSSION THREAD

**Discussion by Okorin** [ppy/osu#15083] [Ideas] [answered]
I would only go there to that tab if i want to move around a timing point, so if this is the situation:
![ex1](https://i.imgur.com/7j2L2X3.png)
i want to move the 1.5x timing to the current timestamp. selecting the difficulty * 1.50x  on the timing panel so i can use the 'Use current time' button will move the timeline in the bottom to the timing of the selected point, effectively rendering what i wanted to do moot

![ex2](https://i.imgur.com/DL9YoT8.png)

only way i have found to get around this is just adding more lines at the current time because selecting any of them on the timing panel will make the button useless...?

I think this behavior should only happen when you... idk double click it or something. just selecting the timing point shouldn't move the timeline i think?

**Comment by peppy** [comment, marked as answer]
difficulty points have already been moved out of that screen, and it is not the final UX either. things will change. i'm going to leave it seeking to the point for now as that's the most common thing for navigating around for actual timing points.

**Reply by Okorin** [reply]
that's good. It was super tedious to deal with
